handlers/handler-flowcontrol-qps/src/main/java/org/apache/servicecomb/qps/ConsumerQpsFlowControlHandler.java

```diff
@@ -38,9 +47,20 @@ public void handle(Invocation invocation, AsyncResponse asyncResp) throws Except
     }
 
     QpsStrategy qpsStrategy = qpsControllerMgr.getOrCreate(invocation.getMicroserviceName(), invocation);
+    String name = qpsStrategy.name();
     if (qpsStrategy.isLimitNewRequest()) {
+      long tps = 0;
+      if ("FixedWindow".equals(name)) {
+        FixedWindowStrategy windowStrategy = (FixedWindowStrategy) qpsControllerMgr.getGlobalQpsStrategy();
+        tps = windowStrategy.getRequestCount().longValue() - windowStrategy.getLastRequestCount() + 1;
+      } else {
+        LeakyBucketStrategy bucketStrategy = (LeakyBucketStrategy) qpsControllerMgr.getGlobalQpsStrategy();
+        tps = bucketStrategy.getRequestCount().longValue();
+      }
       // return http status 429
-      CommonExceptionData errorData = new CommonExceptionData("rejected by qps flowcontrol");
+      LOGGER.warn("consumer qps flowcontrol open, qpsLimit is {} and tps is {}", qpsLimit, tps);
```

Comment:
Log logic basis? A warn level log is a generic error that does not affect business logic.